src/ventus/wm/window.js

```diff
@@ -486,8 +489,23 @@ function(Emitter, Promise, View, WindowTemplate) {
 		},
 
 		move: function(x, y) {
-			this.x = x;
-			this.y = y;
+			var maxX = window.innerWidth - this._margin.right;
+			var maxY = window.innerHeight - this._margin.bottom;
```

Comment:
Where else does Ventus allow the end developer to choose where the windows are? As far as I can tell I dont. That is the whole point of this commit to allow the end developer to not use the whole screen.